pom.xml

```diff
@@ -818,7 +818,7 @@
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
-                <version>0.8.8</version>
+                <version>0.8.12</version>
```

Comment:
This is enough with 0.8.11
Java 21 support is in that release

Reply:
May as well update to $LATEST